contracts/modules/LoanOpenings/LoanOpenings.sol

```diff
@@ -13,55 +13,42 @@ import "../../mixins/InterestHandler.sol";
 import "../../swaps/SwapsUser.sol";
 import "../../governance/PausableGuardian.sol";
 
-
 contract LoanOpenings is State, LoanOpeningsEvents, VaultController, InterestHandler, SwapsUser, PausableGuardian {
-
-    function initialize(
-        address target)
-        external
-        onlyOwner
-    {
+    function initialize(address target) external onlyOwner {
         _setTarget(this.borrowOrTradeFromPool.selector, target);
```

Comment:
set current sig for this function to 0 address on target

Reply:
fixed